src/lib/tsconfig-build.json

```diff
@@ -5,7 +5,7 @@
     "declaration": true,
     "stripInternal": false,
     "experimentalDecorators": true,
-    "noUnusedParameters": true,
+    "noUnusedParameters": false,
```

Comment:
Why are we no longer having this check? Maybe add a comment if necessary

Reply:
Was just trying something during development; reverted